src/content/docs/logs/get-started/logging-best-practices.mdx

```diff
@@ -0,0 +1,158 @@
+---
+title: Logging best practices guide
+tags:
+ - New Relic solutions
+ - Best practices guides
+ - Logs
+ - Logging
+translate:
+ - jp
+metaDescription: 'Best practices for using New Relic logs'
+redirects:
+---
+
+import parsingExample from 'images/logs-parsing-example.png'
+import image from 'images/image.png'
+
+Welcome to the New Relic logging best practices guide. Here you'll find recommendations for:
```

Comment:
one thing I think we should try to do here is differentiate what makes this different from the various docs we have on these things. that's one of our big problems with guide content in general. so we could do something like: 

"Our docs contain general reference information on all these topics. This guide will give you some more in-depth recomemendations and best practices for how to optimize your use of our logs features." Something like that

Reply:
Hmm, I feel conflicted about this because I think it's clear/implied (from the title and intro sentence) that these are recommendations for things already in the docs, especially if we include links out to those docs from this guide.